src/components/Comment/Comment.tsx

```diff
@@ -507,24 +499,30 @@ export const Comment = ({
                           onClick={() => setCommentBeingRepliedTo(comment)}
                           className={cx(
                             commentControlsButtonStyles,
-                            removePaddingLeft
+                            removePaddingLeft,
+                            flexRowStyles
                           )}
                         >
-                          <div className={flexRowStyles}>
-                            <ReplyArrow />
-                            Reply
-                          </div>
+                          <ReplyArrow />
+                          Reply
                         </button>
                       ) : (
-                        <Row>
-                          <ReplyArrow />
-                          <a
-                            className={linkStyles(pillar)}
+                        <div className={commentControlsButtonStyles}>
```

Comment:
`commentControlsButtonStyles` seems to being applied twice here. And actually, now that we're using link, a lot of those styles are redundant